test/tls.js

```diff
@@ -116,6 +116,26 @@ describe('TLS', () => {
     })
   })
 
+  it('should not timeout when it tls connects', done => {
```

Comment:
connects or reconnects?

Reply:
it was on the connect - the first timer set.